index.d.ts

```diff
@@ -49,7 +55,10 @@ export type HotkeyOverride = Record<string, string>
  * ])
  * ```
  */
-export function hotkeyKeyUX(overrides?: HotkeyOverride): KeyUXModule
+export function hotkeyKeyUX(
+  overrides?: HotkeyOverride,
```

Comment:
Hm, can be `overrides` a transformer too?

```js
hotkeyKeyUX([overrides(config), macCompact])
```

What do you think?

Reply:
Yes, I mentioned that earlier: https://github.com/ai/keyux/pull/15#discussion_r1521244882 :)

Since it looks like you agree, I'll go ahead and implement this.